cmd/kops/create_cluster.go

```diff
@@ -89,6 +89,9 @@ type CreateClusterOptions struct {
 	MasterSecurityGroups []string
 	AssociatePublicIP    *bool
 
+	// Set true to not include SSH keys in instance configuration
+	NoSSHKey bool
```

Comment:
It looks like you don't ever make use of this and also ignore it for updates as you have your own setting there. Can we thus keep this out of the cluster spec? It seems like just unused information in there.

Reply:
Yes looks like I ignored update cluster... I'll take a look at it.

If it's not in the cluster spec it looked like I was unable to use it down the API to exclude the SSHKey configuration from the generated instance groups.

Have I misunderstood this perhaps?

Reply:
I missed your nodeup actions against this, sorry. You are right this needs to be in the spec.

Reply:
I missed your nodeup actions against this, sorry. You are right this needs to be in the spec.

Reply:
👍
Still, a true value is not pushed on update cluster. I'll sort that out.

Reply:
On update it looks like only WriteConfigDeprecated is called and the updated value only ends up in the cluster.spec file.

Should CreateClusterConfig be called from here as well?

Reply:
Uh... @mikesplain or @geojaz probably know that ❓